include/nn/bcat.h

```diff
@@ -0,0 +1,113 @@
+/**
+ * @file bcat.h
+ * @brief BCAT service implementation.
+ */
+
+#pragma once
+
+#include "os.h"
+#include "types.h"
+
+namespace nn {
+
+namespace bcat {
+
+struct DirectoryName {
+    void isValid();
+};
+
+struct FileName {
+    void isValid();
+};
+
+class DeliveryCacheDirectory {
+public:
+    DeliveryCacheDirectory();
+    ~DeliveryCacheDirectory();
+    Result Open(nn::bcat::DirectoryName const&);
+    Result GetCount();
+    Result Close();
+};  // namespace DeliveryCacheDirectory
```

Comment:
whoops, I actually missed this... the comment should be removed because it isn't actually a namespace